spec/neo4j-core/unit/query_spec.rb

```diff
@@ -639,6 +639,14 @@ def self.it_generates(cypher, params = {})
     describe '.create(q: {Person: {age: nil, height: 70}})' do
       it_generates 'CREATE (q:`Person` {age: {q_Person_age}, height: {q_Person_height}})', q_Person_age: nil, q_Person_height: 70
     end
+
+    describe ".create(q: {:'Child:Person' => {age: 41, height: 70}})" do
+      it_generates 'CREATE (q:`Child`:`Person` {age: {q_Child_Person_age}, height: {q_Child_Person_height}})', q_Child_Person_age: 41, q_Child_Person_height: 70
+    end
+
+    describe ".create(:'Child:Person' => {age: 41, height: 70})" do
+      it_generates 'CREATE (:`Child`:`Person` {age: {Child_Person_age}, height: {Child_Person_height}})', Child_Person_age: 41, Child_Person_height: 70
+    end
```

Comment:
I'm torn.  I feel like if you specify a key of `:'Child:Person'` that you it should be specifying a label which is `Child:Person`.  It is legal for a label to have a colon in it.

One way that would perhaps be better is to specify an `Array` as the key like this:

`.create([:Child, :Person] => {age: 41, height: 70})`

That could even be:

`.create(%i(Child Person) => {age: 41, height: 70})`

Thoughts?